draft-ietf-quic-multipath.md

```diff
@@ -363,8 +363,8 @@ All new frames are sent in 1-RTT packets {{QUIC-TRANSPORT}}.
 
 ## Path Initiation {#path-initiation}
 
-Opening a new path requires the
-use of a new connection ID (see {{Section 9.5 of QUIC-TRANSPORT}}).
+Opening a new path requires the use of a new connection ID
+for an unused Path ID (see {{explicit-path-id}}).
```

Comment:
```suggestion
Opening a new path requires the use of a new connection ID (see {{Section 9.5 of QUIC-TRANSPORT}})
for an unused Path ID (see {{explicit-path-id}}).
```
I think it is actually not correct to remove the reference to section 9.5 of RFC9000 as that section explain the privacy implications and we don't have a reference to 9.5 of RFC9000 in the section on explicit path ID. However, I think we can have both references.